packages/radio-group/src/Radio.tsx

```diff
@@ -62,33 +77,42 @@ const labelColorSet = {
 
 const labelStyle = css`
   display: flex;
-  align-items: flex-start;
-  font-size: 14px;
   line-height: 20px;
   cursor: pointer;
+  align-items: flex-start;
+  font-size: 13px;
+  font-weight: 700;
 `;
 
-// Note colors are not in our palette
 const inputColorSet = {
   [Mode.Light]: css`
     &:checked + ${inputDisplayWrapper.selector} ${inputDisplay.selector} {
-      background-color: #2798bd;
-      border-color: #2798bd;
+      background-color: ${palette.blue.base};
+      border-color: ${palette.blue.base};
 
       &:after {
         transform: scale(1);
       }
     }
 
+    &:focus-visible + div div {
+      box-shadow: 0 0 0 2px ${palette.white}, 0 0 0 4px ${palette.blue.light1};
+    }
+
```

Comment:
is s possible to target the element itself (maybe using a data selector?). I think we try and avoid this kind of selector if possible